app/models/organizations/business_line.rb

```diff
@@ -166,13 +165,26 @@ def people_join
       "LEFT JOIN people ON claimants.participant_id = people.participant_id"
     end
 
-    # The NUMBER_OF_SEARCH_FIELDS constant reflects the number of searchable fields here for where interpolation later
+    # These values reflect the number of searchable fields in search_all_clause for where interpolation later
+    def number_of_search_fields
```

Comment:
```suggestion
    def number_of_search_fields
      FeatureToggle.enabled?(:decision_review_queue_ssn_column, user: :current_user) ? 4 : 2
    end
```

What do you think of using a ternary operation here?

Reply:
I don't mind it